### DISCUSSION TITLE
New to Open Source - How to get started?

### DISCUSSION THREAD

**Discussion by Jagrit29** [usememos/memos#3900] [Q&A] [answered]
Hi Developers! 
Sorry if this is not the right forum to ask question, etc. But I am new to open source but I wanted to contribute and learn to some tool in Go as that's what I use at work.  I really found this tool nice and it's feels like treasure for learning etc. 

So I was thinking of going through the codebase, reading docs, understanding different parts and then come to open issues and see if I can pick up something accordingly. 

I just wanted to ask if the above sounds good or there is some other way? Thanks in advance!

**Comment by RoccoSmit** [comment, marked as answer]
Hi @Jagrit29, welcome

The below is from the Memos discord (https://discord.gg/tfPJa4UmAv)
![image](https://github.com/user-attachments/assets/beb33b50-e296-4482-afda-aa1c9b6227bd)
(https://discord.com/channels/1017113925776588902/1045138348165050409/1269985731477241917)

**Reply by Jagrit29** [reply]
Amazing, Thanks @RoccoSmit!
